app/assets/stylesheets/partials/_start_page.scss

```diff
@@ -0,0 +1,104 @@
+// Place all the styles related to the StartPage controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+$highlight: 0 1px 0 #fff;
+
+.navbar-nav {
+  margin-top: 15px !important;
+}
+
+.start-background {
+  width: 100%;
+  max-height: 500px;
+  background: image-url("img/bg.jpg") 50% 0%;
+
+  h1 {
+    display: inline-block;
+    margin: 240px 0 0 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `240px 0 0` instead of `240px 0 0 0`